alchemist-full/src/test/resources/remove.yml

```
@@ -0,0 +1,13 @@
+incarnation: scafi
+
+_remover: &remover
+  - time-distribution: 1
+    type: Event
+    actions:
+      - type: RemoveNode
+
+deployments:
+  type: Grid
+  parameters: [-5, -5, 5, 5, 0.5, 0.5, 0.25, 0.25]
```

```suggestion
  parameters: [-1, -1, 1, 1, 0.5, 0.5, 0.25, 0.25]
```

Do we need such a big network? Using something smaller would speed up the tests.